API/Test/test_verify_creating_list_of_users.py

```diff
@@ -0,0 +1,36 @@
+import time
+import pytest
+import requests
+from configuration import BASE_URL
+
+@pytest.fixture
+def user_list():
+    return [
+        {
+            "id": 2,
+            "username": "user1_" + str(int(time.time())),
+            "firstName": "Alice",
+            "lastName": "Smith",
+            "email": "alice.smith@example.com",
+            "password": "password123",
+            "phone": "1234567891",
+            "userStatus": 1
+        },
+        {
+            "id": 3,
+            "username": "user2_" + str(int(time.time())),
+            "firstName": "Bob",
+            "lastName": "Jones",
+            "email": "bob.jones@example.com",
+            "password": "password123",
+            "phone": "1234567892",
+            "userStatus": 1
+        }
+    ]
```

Comment:
It could be stored in constants or dataclass and converted to @pytest.mark.parametrize

Reply:
converted to @pytest.mark.parametrize and moved to constants.py